pkg/sql/grant_revoke.go

```diff
@@ -217,7 +238,8 @@ func (n *changePrivilegesNode) startExec(params runParams) error {
 			}
 
 			for _, grantee := range n.grantees {
-				n.changePrivilege(privileges, n.desiredprivs, grantee)
+				changed := n.changePrivilege(privileges, n.desiredprivs, grantee)
```

Comment:
I don't think it'd be wrong to just have one top-level `changed` and then in all the call sites just:

```
changed = changed || 
   ...
```